1-exercises/I-string-replace/exercise.js

```diff
@@ -1,20 +1,31 @@
-/* 
+/*
   You are given a sentence contains a story.
 
-  Current it says 
+  Current it says
 
   "I like dogs. One day I went to the park and I saw 10 dogs. It was a great day."
 
-  Change the story using .replace() so that it says 
+  Change the story using .replace() so that it says
 
   "I like cats. One night I went to the park and I saw 100000 cats. It was a brilliant night."
 */
 
 let story =
   "I like dogs. One day I went to the park and I saw 10 dogs. It was a great day.";
 
-let result = story.replace("", "");
-
+// let result = story.replace("", "");
+//💫
+let result = story.replace(/dogs|cats|day|night|10|great/gi, (str) =>
+  str === "dogs"
+    ? "cats"
+    : str === "cats"
+    ? "dogs"
+    : str === "day"
+    ? "night"
+    : str === "10"
+    ? "100000"
+    : "brilliant"
+);
```

Comment:
This works well. You don't need the second condition :str==="cats". You could also use multiple, chained replace() methods, but your method is fine.